nixos/modules/config/pulseaudio.nix

```diff
@@ -228,6 +228,9 @@ in {
           source = writeText "libao.conf" "default_driver=pulse"; }
       ];
 
+      environment.sessionVariables.LD_LIBRARY_PATH =
+        [ "${getLib overriddenPackage}/lib" ];
```

Comment:
I don't think using `LD_LIBRARY_PATH` is a good idea for achieving this. In systemd jobs and when the environment gets cleared/filtered (like with `sudo`), `LD_LIBRARY_PATH` won't be inherited and has to be set manually.

Have you explored a solution like mesa-drivers in `/run/opengl-drivers`? The problem with that approach is that NixOS will be needed for PA-based applications. Maybe the RUNPATH can be patched so they will fall back to `libcardiacarrest`?